Plugins/Flow.Launcher.Plugin.BrowserBookmark/Views/SettingsControl.xaml.cs

```diff
@@ -4,119 +4,116 @@
+    {
+        Settings = settings;
+        InitializeComponent();
+    }
 
-        public event PropertyChangedEventHandler PropertyChanged;
+    public event PropertyChangedEventHandler PropertyChanged;
 
-        private void NewCustomBrowser(object sender, RoutedEventArgs e)
-        {
-            var newBrowser = new CustomBrowser();
-            var window = new CustomBrowserSettingWindow(newBrowser);
-            window.ShowDialog();
-            if (newBrowser is not
-                {
-                    Name: null,
-                    DataDirectoryPath: null
-                })
+    private void NewCustomBrowser(object sender, RoutedEventArgs e)
+    {
+        var newBrowser = new CustomBrowser();
+        var window = new CustomBrowserSettingWindow(newBrowser);
+        window.ShowDialog();
+        if (newBrowser is not
             {
-                Settings.CustomChromiumBrowsers.Add(newBrowser);
-                _ = Task.Run(() => Main.ReloadAllBookmarks());
-            }
-        }
-
-        private void DeleteCustomBrowser(object sender, RoutedEventArgs e)
+                Name: null,
+                DataDirectoryPath: null
+            })
         {
-            if (CustomBrowsers.SelectedItem is CustomBrowser selectedCustomBrowser)
-            {
-                Settings.CustomChromiumBrowsers.Remove(selectedCustomBrowser);
-                _ = Task.Run(() => Main.ReloadAllBookmarks());
-            }
+            Settings.CustomChromiumBrowsers.Add(newBrowser);
+            _ = Task.Run(() => Main.ReloadAllBookmarks());
         }
-        
-        private void MouseDoubleClickOnSelectedCustomBrowser(object sender, MouseButtonEventArgs e)
+    }
+
+    private void DeleteCustomBrowser(object sender, RoutedEventArgs e)
+    {
+        if (CustomBrowsers.SelectedItem is CustomBrowser selectedCustomBrowser)
         {
-            EditSelectedCustomBrowser();
+            Settings.CustomChromiumBrowsers.Remove(selectedCustomBrowser);
+            _ = Task.Run(() => Main.ReloadAllBookmarks());
         }
-        
-        private void Others_Click(object sender, RoutedEventArgs e)
-        {
+    }
 
-            if (CustomBrowsersList.Visibility == Visibility.Collapsed)
-            {
-                CustomBrowsersList.Visibility = Visibility.Visible;
-            }
-            else
-                CustomBrowsersList.Visibility = Visibility.Collapsed;
-        }
+    private void MouseDoubleClickOnSelectedCustomBrowser(object sender, MouseButtonEventArgs e)
+    {
+        EditSelectedCustomBrowser();
+    }
 
-        private void EditCustomBrowser(object sender, RoutedEventArgs e)
+    private void Others_Click(object sender, RoutedEventArgs e)
+    {
+        CustomBrowsersList.Visibility = CustomBrowsersList.Visibility switch
         {
-            EditSelectedCustomBrowser();
-        }
+            Visibility.Collapsed => Visibility.Visible,
+            _ => Visibility.Collapsed
+        };
+    }
```

Comment:
In my opinion this is a little easier to read than the original if/else. What do you think?